app/src/main/kotlin/com/simplemobiletools/applauncher/helpers/DBHelper.kt

```diff
@@ -46,7 +46,9 @@ class DBHelper private constructor(val context: Context) : SQLiteOpenHelper(cont
             "CREATE TABLE $MAIN_TABLE_NAME ($COL_ID INTEGER PRIMARY KEY AUTOINCREMENT, $COL_NAME TEXT, $COL_PKG_NAME TEXT UNIQUE, $COL_POSITION INTEGER," +
                 "$COL_WAS_RENAMED INTEGER, $COL_APP_ORDER INTEGER)"
         )
-        addInitialLaunchers(db)
+        if (context.addDefaultApps()) {
```

Comment:
we dont need the helper function, lets just directly check the resources value

Reply:
[removed :+1: ](https://github.com/SimpleMobileTools/Simple-App-Launcher/pull/145/commits/b3143245ba9f85051fcd8cd9e73074842a40962a)